ee/cis/macos-13/cis-policy-queries.yml

```diff
@@ -744,6 +744,27 @@ spec:
 ---
 apiVersion: v1
 kind: policy
+spec:
+  name: CIS - Ensure an Administrator Password Is Required to Access System-Wide Preferences (FDA Required)
```

Comment:
Did you mean ` (Fleetd Required)`? (Because of the use of the new `authdb` Orbit table.)

Reply:
If it also requires FDA I've used: ` (Fleetd, FDA Required)`

Reply:
Oh, that's correct, thanks